experimental/dds/tree2/src/feature-libraries/schema-index/schemaSummarizer.ts

```diff
@@ -131,6 +139,7 @@ export class SchemaSummarizer implements Summarizable {
 		// Currently no Fluid handles are used, so just use JSON.parse.
 		const decoded = this.codec.decode(JSON.parse(schemaString));
 		this.schema.update(decoded);
+		this.schemaIndexLastChangedSeq = 0;
```

Comment:
are we sure 0 is a safe number here? should we be calling the supplied collabWindow?

Reply:
This was discussed with @Abe27342 in a convo outside of this PR, but the idea here is that if load is ever called, we can assume that there is no need to upload a blob unless schema data changes, which is why we set it to 0 (which is guaranteed to be below valid sequence numbers and will trigger incremental summarization).

If load is never called, the `schemaIndexLastChangedSeq` will remain undefined which would mean that incremental summarization will not be used. We purposefully only call supplied collabWindow on `afterSchemaChange`, since we only care about the sequence numbers related to schema ops, and not other ones.

Reply:
was just wondering if you should use Number.MinInteger or something.

Reply:
merge-tree and map use 0 as 'universally known' and -1 as 'unacked' special constants (UniversalSequenceNumber / LocalSequenceNumber respectively). IMO tree should probably do the same.